start.py

```diff
@@ -0,0 +1,18 @@
+import argparse
+from datetime import datetime
+
+from task_1 import get_date, WEEKDAYS, MONTH, Tex

```

Comment:
Константы можно было не импортировать, вы же их все равно не используете.
```suggestion
from task_1 import get_date

```